src/Command/MeiliSearchImportCommand.php

```diff
@@ -76,7 +76,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $entitiesToIndex = array_unique($indexes->toArray(), SORT_REGULAR);
         $batchSize = $input->getOption('batch-size');
-        $batchSize = ctype_digit($batchSize) ? (int) $batchSize : $config->get('batchSize');
+        $batchSize = null !== $batchSize && ctype_digit($batchSize) ? (int) $batchSize : $config->get('batchSize');
```

Comment:
Hey @norkunas what do you think about this?

```suggestion
        $batchSize = ctype_digit((string) $batchSize) ? (int) $batchSize : $config->get('batchSize');
```

I think it will be easier to read, since `ctype_digit` will expect a string anyway :)

Reply:
not sure what we would win that way :) for me it's easier to read that way I've proposed :)

Reply:
I don't know if you are aware of the Cyclomatic Complexity? You could check here about it: https://phpmd.org/rules/codesize.html

But the question is, if we use the suggested version, we are adding a little bit of complexity to the overall complexity of the method (which is bad already 😅), by running this code through phpmd for example, you can see what I'm talking:

With the condition without type casting (initial version):
```
bruno@mars ~ phpmd teste.php text ruleset.xml
/test.php:28	The method __construct() has a Cyclomatic Complexity of 1. 
/test.php:35	The method configure() has a Cyclomatic Complexity of 1. 
/test.php:57	The method execute() has a Cyclomatic Complexity of 16. <<<< affected method
/test.php:155	The method formatIndexingResponse() has a Cyclomatic Complexity of 5. 
```

Without the condition + type casting (suggestion):
```
bruno@mars ~ phpmd teste.php text ruleset.xml
/test.php:28	The method __construct() has a Cyclomatic Complexity of 1. 
/test.php:35	The method configure() has a Cyclomatic Complexity of 1. 
/test.php:57	The method execute() has a Cyclomatic Complexity of 15. <<<< affected method
/test.php:155	The method formatIndexingResponse() has a Cyclomatic Complexity of 5. 
```

By the way phpmd could be really helpful in this repository, in this example I set cyclomatic complexity to zero, to check the amount of complexity added by the change.

Reply:
Yes I know about that and I would argue that this adds more complexity than it is, because it's just strange to cast null to an empty string to make just the php happy about calling ctype_digit.

If you want to really avoid complexity I'd suggest then make an importer service and call it from the command :)

Reply:
Hi, @norkunas. I'm sorry for the huge delay. I am still being used to dealing with 30+ integrations 😅.

My main idea with this and the other repositories is to have the help of automated tools to let them review most of the code for my team (integrations team) and me.
When we implement them here, it will start complaining about this addition and many others, so it does not make much sense to accept this contribution since I will change it in the future.

Thanks for your contribution and I hope this does not discourage you!